add Gadi profile (gpuhopper)

BenWibking opened the pull request:
### Description
Adds an environment profile script for compiling/running on Gadi (Hopper) nodes. Tested on Gadi.

Performance measurement for `HydroBlast3D` on 1 gpuhopper node:
```
Performance figure-of-merit: 0.001111289931 μs/zone-update [899.855179 Mupdates/s]
```
(With tuning, it should be possible to get ~2x this performance.)

### Related issues
N/A

### Checklist
_Before this pull request can be reviewed, all of these tasks should be completed. Denote completed tasks with an `x` inside the square brackets `[ ]` in the Markdown source below:_
- [x] I have added a description (see above).
- [x] I have added a link to any related issues (if applicable; see above).
- [x] I have read the [Contributing Guide](https://github.com/quokka-astro/quokka/blob/development/CONTRIBUTING.md).
- [ ] I have added tests for any new physics that this PR adds to the code.
- [ ] *(For quokka-astro org members)* I have manually triggered the GPU tests with the magic comment `/azp run`.

BenWibking committed:
add Gadi profile (gpuhopper)

BenWibking committed:
remove old pbs scripts; add gpuhopper.pbs

BenWibking committed:
update gadi docs

BenWibking committed:
rename

BenWibking committed:
update gadi docs

BenWibking committed:
max_timesteps=1000

BenWibking committed:
Merge branch 'BenWibking/add-gadi-hopper-profile' of github.com:quokka-astro/quokka into BenWibking/add-gadi-hopper-profile

chongchonghe commented:
What's the reason to delete so many pbs scripts? @BenWibking

BenWibking commented:
> What's the reason to delete so many pbs scripts? @BenWibking

They were all old. I didn't think anyone was using them. They all only work for gpuvolta. Since gpuhopper is so much faster, I didn't think they were useful anymore.

chongchonghe commented:
OK, no one that I know is using Gadi V100, so it's fine to remove them.

markkrumholz commented:
I would actually prefer to leave the scripts for gpuvolta in place. We aren't currently using them much, but there is no guarantee that we might not want to run some stuff on Gadi next year, and the only GPU queue with any significant number of cards is gpuvolta.

BenWibking committed:
add gpuvolta/

BenWibking commented:
> I would actually prefer to leave the scripts for gpuvolta in place. We aren't currently using them much, but there is no guarantee that we might not want to run some stuff on Gadi next year, and the only GPU queue with any significant number of cards is gpuvolta.

@markkrumholz I've moved the old PBS scripts to `scripts/pbs/gpuvolta`. I deleted a few scripts that I think were only ever used for debugging.
